settings_example.py

```diff
@@ -0,0 +1,12 @@
+
+"""
+Sample environment file for describing the location of resources external to the repository.
+"""
+
+project_path = '.'
+dirpath = project_path + "/data/"
+vizpath = project_path + "/visualizations_output/"
+fontpath = project_path + "/grls-visualizations/"
```

Comment:
It doesn't seem like we are using the fontpath variable here. Maybe it would be nice if we did? (maybe demo_vis.py:82)